packages/mendel-mocha-runner/index.js

```diff
@@ -36,7 +36,7 @@ function MendelRunner(filePaths, options={}) {
 
         const mocha = new Mocha(options);
 
-        const entries = client.registry.getEntriesByGlob(filePaths);
+        const entries = client.registry.getEntriesByGlob(filePaths, false);
```

Comment:
See suggestion about not filtering. But if we keep the filtering, can we have meaningful options instead, even if this is the only config so far?

```js
// usage
client.registry.getEntriesByGlob(filePaths,{includeVariations:true});
// implementation
getEntriesByGlob(globStrings, options = {includeVariations:false}) {/*...*/}
```